apps/music/test/marionette/loading_test.js

```diff
@@ -0,0 +1,76 @@
+/* global require, marionette, setup, suite, test, __dirname */
+'use strict';
+
+var assert = require('assert');
+var Music = require('./lib/music.js');
+var FakeRingtones = require('./lib/fakeringtones.js');
+var FakeControls = require('./lib/fakecontrols.js');
+
+marionette('Music loading', function() {
+  var apps = {};
+  apps[FakeRingtones.DEFAULT_ORIGIN] = __dirname + '/fakeringtones';
+  apps[FakeControls.DEFAULT_ORIGIN] = __dirname + '/fakecontrols';
+
+  var client = marionette.client({
+    prefs: {
+      'device.storage.enabled': true,
+      'device.storage.testing': true,
+      'device.storage.prompt.testing': true
+    },
+
+    settings: {
+      'lockscreen.enabled': false,
+      'ftu.manifestURL': null
+    },
+
+    apps: apps
+  });
+
+  var music;
+
+  setup(function() {
+    music = new Music(client);
+
+  });
+
+  suite('Detection of movies.', function () {
+    setup(function() {
+      client.fileManager.removeAllFiles();
+      client.fileManager.add([
+        {
+          type: 'music',
+          filePath: 'apps/music/test-data/playlists/a.ogg'
+        },
+        {
+          type: 'music',
+          filePath: 'test_media/Movies/gizmo2.mp4'
+        }
+      ]);
+    });
+
+    test('Check the mp4 isn\'t loaded', function() {
+
+      music.launch();
+      music.waitForFirstTile();
+
+      music.switchToAlbumsView();
+
+      music.selectAlbum('We crash computers');
+
```

Comment:
If you select just one album, then of course the movie file won't show up there, since it is not in the same album as the song. I think you need to switch to show the list of all songs and then verify that there is only one song in that list, right?